packages/mui-material/src/Checkbox/Checkbox.js

```diff
@@ -231,12 +250,9 @@ Checkbox.propTypes /* remove-proptypes */ = {
   indeterminateIcon: PropTypes.node,
   /**
    * [Attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Attributes) applied to the `input` element.
+   * @deprecated Use `slotProps.input` instead. This prop will be removed in v7. See [Migrating from deprecated APIs](/material-ui/migration/migrating-from-deprecated-apis/) for more details.
    */
   inputProps: PropTypes.object,
-  /**
```

Comment:
@siriwatknp Is this expected?

Reply:
Yes, those are generated from `SwitchBase`. Once this PR is merged, I will merge the #44972 right away.

Reply:
Okay then, I'll merge PR